scripts/fuzz_opt.py

```diff
@@ -1195,10 +1199,17 @@ def randomize_opt_flags():
 ^^^^
 ||||
 
-Make sure to verify by eye that the output says
+Make sure to verify by eye that the output says something like this:
 
-should be 0: 0
-should be 1: 1
+At least one of the next two values should be 0:
+  0
+  1
+The following value should be 1:
+  1
+
+(If it does not, then one possible issue is that -ttf fails to write a valid
```

Comment:
I don't remember what -ttf stands for. Could we use the full length option name here instead?